code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -420,7 +420,7 @@
 				to_chat(Carbies, "<span class='danger'>You feel your burns and bruises healing! It stings like hell!</span>")
 			SEND_SIGNAL(Carbies, COMSIG_ADD_MOOD_EVENT, "painful_medicine", /datum/mood_event/painful_medicine)
 			//Has to be at less than TRESHOLD_UNHUSK burn damage and have 100 isntabitaluri before unhusking. Corpses dont metabolize.
-			if(HAS_TRAIT_FROM(M, TRAIT_HUSK, "burn") && Carbies.getFireLoss() < TRESHOLD_UNHUSK && Carbies.reagents.has_reagent(/datum/reagent/medicine/C2/instabitaluri, 100))
+			if(HAS_TRAIT_FROM(M, TRAIT_HUSK, "burn") && Carbies.getFireLoss() < TRESHOLD_UNHUSK && Carbies.reagents.has_reagent(/datum/reagent/medicine/C2/instabitaluri, SYNTHFLESH_HUSKFIX_THRESHOLD)) //FULPSTATION MEDBORG CHANGES -Surrealistik Feb 2020
```

Comment:
you might want to incorporate the synthflesh changes that I made in this PR (notably, I fixed an important oddity/bug with synthflesh patches): https://github.com/tgstation/tgstation/pull/49545

Reply:
What's the fix exactly? I'd rather not dig

Reply:
there are quite a few, and they're all listed in the OP of the PR

Reply:
among several other things, without the changes from that PR, 40u synthflesh patches from your PR won't unhusk people with one patch

Reply:
Check the code updates.

Also I'm not bothering with the name changes and such due to conflicts; changes will be kept minimal vis a vis TG; only what is strictly and mechanically necessary.

Reply:
wait, how would NOT changing the name to match the /tg/ version cause a merge conflict?